matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/RedactEventWorker.kt

```diff
@@ -43,25 +45,34 @@ internal class RedactEventWorker(context: Context, params: WorkerParameters, ses
             val roomId: String,
             val eventId: String,
             val reason: String?,
+            val withRelations: List<String>? = null,
             override val lastFailureMessage: String? = null
     ) : SessionWorkerParams
 
     @Inject lateinit var roomAPI: RoomAPI
     @Inject lateinit var globalErrorReceiver: GlobalErrorReceiver
+    @Inject lateinit var homeServerCapabilitiesService: HomeServerCapabilitiesService
```

Comment:
Should we inject `RedactEventTask` instead and use it here to redact the event?

Reply:
(But `RedactEventWorker` is not used anymore...)

Reply:
Yes I think, good catch! https://github.com/vector-im/element-android/pull/7988/commits/a3f65b9bcbaba62f88c983a5ff8f5a3821ae11ed